Tensile/Common.py

```diff
@@ -1579,21 +1580,28 @@ def assignParameterRequired(destinationDictionary, key, sourceDictionary):
 # Push / Pop Working Path
 # store a WorkingPath where to write files (like benchmark files)
 ################################################################################
+def ensurePath( path ):
```

Comment:
Moving this function here puts the comment above in the wrong place.

Reply:
Resolved in last commit